group/assignment5/main.cpp

```diff
@@ -0,0 +1,96 @@
+#include "AlphabetFromDictionary.hpp"
+#include "codeu_test_lib.h"
+
+int main() {
+    vector<string> dictionary;
+    vector<char> alphabet;
+    bool result;
+    unordered_map<char, int> idxs;
+    vector<bool> expectedChars;
+    vector<bool> actualChars;
+
+    // test 1 - assignment example
+    dictionary = {"ART", "RAT", "CAT", "CAR"};
+    AlphabetFromDictionary afd(dictionary);
```

Comment:
When I saw _afd_ in this test, by its name I could not guess what it is supposed to be. I think that it is better to have long name alphabetFromDictionary, so reader will know meaning of it.

Reply:
Sure, we can change that, I don't mind.